src/Components/SmartComponents/SystemsPage/SystemsPage.js

```diff
@@ -112,22 +110,18 @@ const SystemsPage = ({ intl }) => {
         if (selectedHosts && (selectedHosts.length > 0) || systemId) {
             dispatch(optOutSystemsAction(systemId || selectedHosts, optOut)).then(() => {
                 dispatch(fetchSystems({ ...parameters, page: 1 }));
-                parameters.opt_out === false
-                    ? dispatch(addNotification({
-                        variant: 'success',
-                        title: intl.formatMessage(
-                            messages.systemsExcludedNotification,
-                            { count: systemId ? 1 : selectedHosts.length || 0 }
-                        )
-                    }))
-                    : dispatch(addNotification({
-                        variant: 'success',
-                        title: intl.formatMessage(
-                            messages.systemsResumedNotificationTitle,
-                            { count: systemId ? 1 : selectedHosts.length || 0 }
-                        ),
+
+                let count = systemId ? 1 : selectedHosts.length || 0;
+
+                if (optOut) {
+                    addNotification({ msg: intl.formatMessage(messages.systemsExcludedNotification, { count }) });
```

Comment:
I really don't like how we need to format the message and pass the values, like `{....}` it looks so ugly :(

Reply:
Maybe a custom hook which will act like a "wrapper". Also I don't like the idea of importing `messages` on every single page. 

something like 
```   
  const formatIntl = useCustomIntl();
  formatIntl(id = string, values = object)


  ----------------
  
   addNotification({ msg: formatIntl(systemsExcludedNotification, {values}) });
```
hmm, we don't get rid off `{values}`

Reply:
Well sometimes knowing `{values}` right away can be useful, for example 
`intl.formatMessage(messages.lastxdays, { days: 7 })`
so I don't think we could get rid of it, but I'm in for shortening `intl.formatMessage(messages` to something more reasonable with custom wrapper.